src/index.js

```diff
@@ -0,0 +1,273 @@
+/* eslint no-use-before-define: 0, no-extra-parens: 0 */
+type StateOpts = {
+  cssmodule: string;
+  modules: "commonjs"|"es6";
+};
+
+type State = {
+  opts: StateOpts
+};
+
+import template from "babel-template";
+import find from "array-find";
+
+/**
+ * TemplateElement value nodes must be of the shape {raw: string; value: string}
+ */
+const templateElementValue = (value) => ({raw: value, cooked: value});
+
+
+export default ({types: t}) => {
+  const ROOT_CSSNAMES_IDENTIFIER = "cssmodule";
+
+  const matchesPatterns = (path, patterns) => (
+    !!find(patterns, (pattern) => (
+      t.isIdentifier(path.node, { name: pattern }) ||
+      path.matchesPattern(pattern)
+    ))
+  );
```

Comment:
So you're effectively using () as {} to avoid typing the work return... This took me a minute to figure out but I get it. Might be a little too clever for my taste but It's definitely interesting :)